lighthouse-core/gather/driver.js

```diff
@@ -506,12 +557,19 @@ class Driver {
       options: 'sampling-frequency=10000'  // 1000 is default and too slow.
     };
 
-    // Disable any domains that could interfere or add overhead to the trace
-    return this.sendCommand('Debugger.disable')
-      .then(_ => this.sendCommand('CSS.disable'))
-      .then(_ => this.sendCommand('DOM.disable'))
-      // Enable Page domain to wait for Page.loadEventFired
-      .then(_ => this.sendCommand('Page.enable'))
+    // Check any domains that could interfere with or add overhead to the trace.
+    if (this.domainEnabled('Debugger')) {
```

Comment:
I'm specifically thinking of CRX users who wouldn't see cli/console logs. But if the error surfaces up at the end (or as a failure during the CRX run), then it's all good.

> assert the domains are disabled and throw if they aren't

Yea. That would be a good way for use to verify no one tries do so anything crazy during a trace in the future.